packages/config/src/configDefaults.ts

```diff
@@ -62,6 +62,7 @@ export const getInitialConfig = ({
     },
     compilers: {
       solc: {
+        nativePath: `${process.env.TRUFFLE_NATIVE_SOLC_PATH || "solc"}`,
```

Comment:
I snuck this env override w/ my changes, and want to be sure  @eggplantzzz and @haltman-at are ok with it.

Reply:
Eech.  I am quite wary of that... @gnidan?